README.rst

```diff
@@ -1,28 +1,94 @@
-djangocms-googlemap
-===================
+=====================
+django CMS Google Map
+=====================
 
-A Google Maps plugin for django CMS.
+
+|pypi| |build| |coverage|
+
+**django CMS Google Map** is a set of plugins for `django CMS <http://django-cms.org>`_
+that allow you to implement `Google Map <https://developers.google.com/maps/>`_ into your website.
+
+This addon is compatible with `Aldryn <http://aldryn.com>`_ and is also available on the
+`django CMS Marketplace <https://marketplace.django-cms.org/en/addons/browse/djangocms-googlemap/>`_
+for easy installation.
+
+.. image:: preview.gif
+
+
+Contributing
+============
+
+This is a an open-source project. We'll be delighted to receive your
+feedback in the form of issues and pull requests. Before submitting your
+pull request, please review our `contribution guidelines
+<http://docs.django-cms.org/en/latest/contributing/index.html>`_.
+
+One of the easiest contributions you can make is helping to translate this addon on
+`Transifex <https://www.transifex.com/projects/p/djangocms-googlemap/>`_.
+
+
+Documentation
+=============
+
+See ``REQUIREMENTS`` in the `setup.py <https://github.com/divio/djangocms-googlemap/blob/master/setup.py>`_
+file for additional dependencies:
+
+* Python 2.7, 3.3 or higher
+* Django 1.8 or higher
 
 
 Installation
 ------------
 
-This plugin requires `django CMS`_ 3.0 or higher to be properly installed.
+For a manual install:
```

Comment:
OK, but at some point we should correct these errors, rather than just keep them because they've been made already!